lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -551,17 +569,18 @@ mapping:
 
       builds_file_path:
         type: str
-        default: tool-data/shared/ucsc/builds.txt
+        default: shared/ucsc/builds.txt
         required: false
         desc: |
-          File containing old-style genome builds
+          File containing old-style genome builds. Value will be resolved to
+          '<tool_data_path>/shared/ucsc/builds.txt'.
 
       len_file_path:
         type: str
-        default: tool-data/shared/ucsc/chrom
```

Comment:
Add:
```yaml
        default: shared/ucsc/chrom
```

Reply:
well, that was stupid. Thanks. Done, pushing.